### PR TITLE
My Units Page `/profile` and Account Menu

### PR DESCRIPTION
**Pull request by ragizaki**
This PR creates the my units page at `/profile`, where users can see all their created units, as well as delete and edit their units. The page will show a simple text message if the user does not have any posts created. The profile link is accessible when the user is logged in, from the account menu in the navbar.

I created a simple account menu for users, that shows links for logging out and visiting my units.

Currently, the delete and edit buttons don't have any functionality, that'll be from Martin and Jenny's PR and we'll just copy that logic over to this page.

## Acceptance Criteria
- Account Menu only shows when a user is logged in, otherwise the Login and Signup links are shown
- The first letter of the user's username is shown as the Account Menu avatar
- Clicking on the account menu and clicking logout will successfully logout a user as before
- Clicking on my units in account menu will redirect to the `/profile` page
- Creating a unit will show that unit on the user's profile page

### CODE REVIEW

File: api/app.py [hunk 1]
```diff
@@ -173,6 +173,30 @@ def get_units():
...
+   
+    for r in rv:
+        file_name = basedir + r['image_path']
+        img = cv2.imread(file_name)
+        jpg_img = cv2.imencode('.jpg',img)
+        b64_string = base64.b64encode(jpg_img[1]).decode('utf-8')
+        r["image_data"] = b64_string
+    rv = tuple(rv)
+    return {"data": rv}
+
```
Comment (raveen-singh) [inline api/app.py:199]: Why is this here? It looks like it already is in main and this is a duplicate. Please remove if so.

**Review by raveen-singh — CHANGES_REQUESTED** [review]
Everything works as described. The FE is good, small thing to look at in the BE.

File: api/app.py (hunk 1 above)
Comment (ragizaki) [reply api/app.py:199]: I copied this from the other route because without it I can't display the image without it

Comment (ragizaki) [reply api/app.py:199]: also the other route is for a single unit, this one is for a user's units, the routes look really similar tho

**Comment by jennybai01** [comment]
It would be nice if, after logging in, user is directed to my units.

**Comment by jennybai01** [comment]
lgtm otherwise!

File: api/app.py (hunk 1 above)
Comment (raveen-singh) [reply api/app.py:199]: Oh I see you're right, mb.

**Review by raveen-singh — APPROVED** [review]
Lgtm. Please merge.
